ruby/vampires.rb

```diff
@@ -1,4 +1,4 @@
-puts "How many employees will be processed?"
+puts "How many employees would you like to process?"
 applicants = gets.chomp
 
 current_year = 2016

```

Comment:
with this setup you will have to change this file every year for it to work.

Check out the Ruby `Time` class. You should be able to handle a dynamic year check with something like `Time.new.year`